job.go

```diff
@@ -102,13 +104,37 @@ func (job *Job) withNotify(progress int, f func() error) func() error {
 	}
 }
 
-func (job *Job) setupWorkspace(ctx context.Context, f func() error) error {
+func (job *Job) prepare() error {
+	err := job.message.Validate()
+	if err != nil {
+		log.Printf("Invalid Message: MessageId: %v, Message: %v, error: %v\n", job.message.MessageId(), job.message.raw.Message, err)
+		return nil
```

Comment:
Is `return nil` right? (If so, validation error is ignored and `job.setupWorkspace()`, `job.setupDownloadFiles()`, `job.build()` are skipped.)
Maybe, it should be `return err`.